backend/ml/tests/test_mehestan_primitives.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+from django.test import TestCase
+
+from ml.mehestan.primitives import QrMed, QrUnc
+
+
+class QrMedTest(TestCase):
+    def test_qrmed(self):
+        W = 2
+        weight = 1
+        self.assertAlmostEqual(
+            QrMed(
+                W,
+                w=weight,
+                x=np.array([-10.0, 1.0, 10.0]),
+                delta=np.array([1e-3, 1e-3, 1e-3]),
+            ),
+            0.5, # influence of the median contributor is bounded by weight / W
```

Comment:
I don't understand why this works. From my understanding of QrMed, it should be somewhere between 0 and 0.5, but not exactly 0.5. It is not like for BrMean where we have a clipping.

Reply:
It is, the exact value is about 0.4999990 (i.e not a rounding error). It illustrates how with low enough uncertainties in `delta`, the result will be arbitrary close to the theoretical bound for individual contributions. And I thought that could be a useful example to show how QrMed behaves like a "regularized median". Would it be less confusing to use more decimals in the test?

I suppose a good task for #1185 could be to define more tests cases to explain the impact of each parameter, and ideally find examples where the expected result would seem relatively intuitive.

Reply:
I just understood QrMed badly, you test looks good.